doc.go

```diff
@@ -1,5 +1,5 @@
 /*
-A WebAssembly runtime for Go powered by Wasmtime.
+Package wasmtime is a WebAssembly runtime for Go powered by Wasmtime.
```

Comment:
This doesn't quite roll off the tongue for me, is there a particular reason this changed?

Reply:
https://golang.org/doc/effective_go.html#commentary

> Every package should have a package comment, a block comment preceding the package clause. For multi-file packages, the package comment only needs to be present in one file, and any one will do. The package comment should introduce the package and provide information relevant to the package as a whole. It will appear first on the godoc page and should set up the detailed documentation that follows.